repose-aggregator/components/filters/client-auth/src/test/groovy/org/openrepose/filters/clientauth/AuthNFilterDeprecationTest.groovy

```diff
@@ -0,0 +1,32 @@
+package org.openrepose.filters.clientauth
+
+import com.mockrunner.mock.web.MockFilterConfig
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.test.appender.ListAppender
+import org.openrepose.core.services.config.ConfigurationService
+import org.openrepose.core.services.datastore.DatastoreService
+import spock.lang.Specification
+
+import static org.mockito.Mockito.mock
+
+class AuthNFilterDeprecationTest extends Specification {
+    def 'logs a deprecation warning upon init'() {
+        given:
+        System.setProperty("javax.xml.parsers.DocumentBuilderFactory",
```

Comment:
SO you are having to explicitly say which one in tests, because some other library is setting it to something else. So part of our code base is expecting the one we use to use, and now something new brought in is probably counting on the one that has made all the other tests die, by changing that value.